test/application/versioning/Release.test.js

```diff
@@ -52,11 +53,14 @@ contract('Release', ([_, developer, anotherAddress, implementation_v0, implement
   })
 
   describe('setImplementation', function () {
+    beforeEach(async function() {
+      this.implementation_v0 = (await DummyImplementation.new()).address
+      this.implementation_v1 = (await DummyImplementation.new()).address
```

Comment:
I think you are not using `this.implementation_v0`. Additionally, I'd remove the `implementation_v0` and `implementation_v1` variables declared above in the `contracts` context block.

Reply:
`this.implementation_v0` is used in `shouldBehaveLikeImplementationDirectory`. It isn't used in the `when it is frozen` but I left it for simplicity.